### DISCUSSION TITLE
Q: 如何配置或者调整登录选项？ || Q: How to configure or adjust login options?

### DISCUSSION THREAD

**Discussion by aiddroid** [walinejs/waline#2485] [Q&A] [answered]
### Describe the bug | 问题描述

如何配置或者调整登录选项？比如只让他支持Facebook 和 Google登录

### Website URL | 问题网站

https://waline-app-orcin.vercel.app/

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

How do I configure or adjust login options? For example, only let him support Facebook and Google login

### Website URL | Problem website

https://waline-app-orcin.vercel.app/

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment, marked as answer]
一般不建议修改这个。如果需要修改，可以将 admin 的页面通过 vercel.json 的 rewrites 的能力映射到自己的 html 文件中，然后注入下 `window.ALLOW_SOCIALS = ['facebook', 'google'];` 的变量吧。



---
It is generally not recommended to modify this. If you need to modify it, you can map the admin page to your own html file through the rewrites capability of vercel.json, and then inject the variable `window.ALLOW_SOCIALS = ['facebook', 'google'];`.
